.github/workflows/goldsky-deploy.yml

```
@@ -0,0 +1,53 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: "npm"
+
+      - name: Install Goldsky CLI
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 25: did not find expected key

(syntax-check)